packages/Php74/tests/Rector/Property/TypedPropertyRector/FixtureUnionTypes/two_types.php.inc

```diff
@@ -18,9 +18,6 @@ namespace Rector\Php74\Tests\Rector\Property\TypedPropertyRector\FixtureUnionTyp
 
 final class TwoTypes
 {
-    /**
-     * @var bool|string
-     */
     private bool|string $unionValue;
```

Comment:
This test seems to be in a wrong namespace. Union types are part of PHP 8.0, not 7.4.

Reply:
This tests property type, not union type